caas/docker/versions.tf

```
@@ -1,7 +1,8 @@
 terraform {
   required_providers {
     datadog = {
-      source = "terraform-providers/datadog"
+      source = "DataDog/datadog"
+      version = ">= 3.1.0"
     }
   }
   required_version = ">= 0.12.26"
```

Didn't you mean to bump this for all modules?

```suggestion
  required_version = ">= 0.12.31"
```

---

yes thanks, should be good